h/static/scripts/group-forms/components/AnnotationCard.tsx

```diff
@@ -100,15 +101,19 @@ export default function AnnotationCard({ annotation }: AnnotationCardProps) {
             </ul>
           )}
 
-          <footer className="flex items-center justify-between">
-            <div />
+          <footer className="flex items-end justify-between">
+            <ModerationStatusSelect
+              onChange={/* istanbul ignore next */ () => {}}
+              selected={annotation.moderation_status}
+              mode="select"
+              alignListbox="left"
+            />
             <div className="flex items-center gap-1">
               <Link
                 variant="text-light"
                 href={annotation.links.incontext}
                 target="_blank"
                 title="See in context"
-                aria-label="See in context"
```

Comment:
I added the a11y test, which I originally overlooked, and it reported links should not have an aria-label if they don't have a role. I removed it and it seems the `title` is good enough, and screen readers announce it.

Reply:
Oooh, good catch.

Yeah, if I have a complain about axe-core is that errors are frequently not super clear